src/olympia/users/utils.py

```diff
@@ -285,3 +288,108 @@ def upload_picture(user, picture):
         user.picture_path,
         set_modified_on=user.serializable_reference(),
     )
+
+
+def assert_socket_labs_settings_defined():
+    if not settings.SOCKET_LABS_TOKEN:
+        raise Exception('SOCKET_LABS_TOKEN is not defined')
+
+    if not settings.SOCKET_LABS_HOST:
+        raise Exception('SOCKET_LABS_HOST is not defined')
+
+    if not settings.SOCKET_LABS_SERVER_ID:
+        raise Exception('SOCKET_LABS_SERVER_ID is not defined')
+
+
+utils_log = olympia.core.logger.getLogger('z.users')
+
+
+def check_suppressed_email_confirmation(verification=None, page_size=5):
+    assert_socket_labs_settings_defined()
+
+    if not verification:
+        raise Exception('Verification is not defined')
```

Comment:
Not sure what this is for - why have `verification` be an optional parameter if we want to raise for it not being provided?

Reply:
Would something like this make more sense? Making it a required argument and assert it is a suppressed email verification?

    assert isinstance(verification, SuppressedEmailVerification)

Reply:
If you were feeling particularly paranoid, I guess.  But unless the `verification` value passed in somehow happened to have a `.suppressed_email.email` attribute it'd raise on the next line anyway.